Update fTPM and Authvar TAs (#54)
ms-iot/imx-iotcore: Documentation/porting-imx.md

- [PWM WinRT APIs](https://docs.microsoft.com/en-us/uwp/api/windows.devices.pwm)
- [Virtual PWM Driver Sample](https://github.com/Microsoft/Windows-iotcore-samples/tree/develop/Drivers)

## Security TAs

> UEFI includes a pair of OP-TEE Trusted Applications (TAs) which implement a firmware TPM, and a UEFI authenticated variable store. These binaries should generally not require re-compiling; however, if your OP-TEE has been changed it may introduce incompatibilities. See [Updating the TAs](./build-firmware.md#Updating-the-TAs) for instructions on adding new TAs to your firmware binaries.

Reviewer: ```suggestion
UEFI includes a pair of OP-TEE Trusted Applications (TAs) which implement a firmware TPM, and a UEFI authenticated variable store. These binaries should generally not require re-compiling; however, if your OP-TEE has been changed (including build flags) it may introduce incompatibilities. See [Updating the TAs](./build-firmware.md#Updating-the-TAs) for instructions on adding new TAs to your firmware binaries.
```

> They are included in UEFI by default but can be omitted with the CONFIG_NOT_SECURE_UEFI=1 flag. The TAs require OP-TEE to have access to secure storage (eMMC's RPMB). Windows will not support Bitlocker or Secure Boot without these TAs enabled.

Reviewer: ```suggestion
They are included in UEFI by default but can be omitted with the CONFIG_NOT_SECURE_UEFI=1 flag. The TAs require OP-TEE to have access to secure storage (eMMC's RPMB). Windows will not support Bitlocker, Secure Boot, or persistent firmware variable storage without these TAs enabled.
```

## Building UEFI

> For a detailed guide on how to build the iMX UEFI firmware image, please refer to [Building and Updating Firmware](./build-firmware.md).

Reviewer: ```suggestion
For a detailed guide on how to build the i.MX UEFI firmware image, please refer to [Building and Updating Firmware](./build-firmware.md).
```
